src/app/models/transmart-models/transmart-data-table.ts

```diff
@@ -1,14 +1,14 @@
 import {TransmartDimension} from './transmart-dimension';
 import {TransmartRow} from 'app/models/transmart-models/transmart-row';
-import {TransmartSorting} from './transmart-sorting';
+import {TransmartSort} from './transmart-sorting';
```

Comment:
try to keep the naming consistent, TransmartSort - transmart-sort.ts

Reply:
ah sorry, I forgot to rename the file, I will update it, thanks!